### PR TITLE
Add functionality to reviews component and basic styling

### PR DESCRIPTION
**Pull request by jessicaawhy**
Added sort and filter functionality for reviews as well as styling to get components arranged on page. Currently, tests are failing & need to be updated.

### CODE REVIEW

File: client/components/Reviews/Reviews.jsx [hunk 1]
```diff
@@ -1,30 +1,58 @@
...
+    border: none;
+    text-decoration: underline;
+  `;
+
   return (
     <div data-testid="reviews">
       <div>
         {`${reviews.length} reviews, sorted by `}
-        <button type="button">relevance</button>
+        {/* eslint-disable-next-line styled-components-a11y/no-onchange */}
```
Comment (oliviaschwerdt) [inline client/components/Reviews/Reviews.jsx:43]: Is this next-line comment something we should add to the .eslintrc, or is it being temporarily used here? No action needs to be taken for this, just something to consider.

**Review by oliviaschwerdt — APPROVED** [review]
Great job on adding all the functionality for your component, and getting the basic styling in! The logic you used for the sorting is clever, and you did a great job of adding styling and relevant tests for your components. I left a few comments just for your consideration, but nothing needs to be changed before merging. Beautiful work!!

File: client/components/Reviews/Summary.jsx [hunk 2]
```diff
@@ -1,45 +1,121 @@
...
+            Current ratings filter:
+            {' '}
+            {currentFilter.join(', ')}
+          </p>
+          <ButtonLink onClick={() => setFilter({})} type="button">Remove all filters</ButtonLink>
+        </div>
+        )
+      }
+      <LinkContainer>
+        {/* a lot of repeated code, can be refactored */}
```
[the remark this answers is not in the document]
Comment (jessicaawhy) [reply client/components/Reviews/Summary.jsx:89]: Will consider / do this during refactoring. Thanks!

File: client/components/Reviews/Summary.jsx [hunk 3]
```diff
@@ -1,45 +1,121 @@
...
+    display: flex;
+    width: 100%;
+    > * {
+      margin: 5px;
     }
+  `;
 
-    return total / count;
-  };
+  const ButtonLink = styled.button`
```
[the remark this answers is not in the document]
Comment (jessicaawhy) [reply client/components/Reviews/Summary.jsx:20]: I  agree with this & think we should discuss it as well.

File: client/components/Reviews/Reviews.jsx (hunk 1 above)
Comment (jessicaawhy) [reply client/components/Reviews/Reviews.jsx:43]: It is temporarily being used. I am going to try to find a way to implement the select options without needing onchange, but we that will be something for later.

**Pull request merged**